win32/build/template.rc

```diff
@@ -64,7 +64,7 @@ BEGIN
 #endif
             VALUE "FileVersion", EXT_VERSION
             VALUE "InternalName", INTERNAL_NAME
-            VALUE "LegalCopyright", "Copyright � 1997-2018 The PHP Group"
+            VALUE "LegalCopyright", "Copyright � The PHP Group"
```

Comment:
UTF-8 conversation?

Reply:
> file win32/build/template.rc
```
win32/build/template.rc: c program text, ISO-8859 text
```
So far is ok... Before it was ISO-8859, which is probably ok for Windows' case. And it is otherwise generated.